modules/core/src/test/java/org/locationtech/jts/geom/prep/PreparedGeometryConcurrencyBugTest.java

```diff
@@ -0,0 +1,67 @@
+package org.locationtech.jts.geom.prep;
```

Comment:
This should go into the `test.jts.perf.geom.prep` package, because:
* it requires custom VM settings
* it is not guaranteed to reveal the issue on every run
* it may require significant execution time

Reply:
fixed.